File: PHP/Program/Shirt.php
<?php

require_once 'Clothing.php';

class Shirt extends Clothing {
    private $color;
    private $sleevetype;

    public function __construct($color = "", $sleevetype = "") {
        parent::__construct(); // Panggil konstruktor dari kelas induk
        $this->color = $color;
        $this->sleevetype = $sleevetype;
    }

    public function get_color() {
        return $this->color;
    }

    public function set_color($color) {
        $this->color = $color;
    }

    public function get_sleevetype() {
        return $this->sleevetype;
    }

    public function set_sleevetype($sleevetype) {
        $this->sleevetype = $sleevetype;
    }

    public function __destruct() {
        
    }
}

?>

File: CPP/Program/Shirt.cpp
#include <iostream>
#include <string>
#include "Clothing.cpp"

using namespace std;

class Shirt : public Clothing
{
    private: 
        string color;
        string sleevetype;

    public:
        Shirt()
        {
            this->color = "";
            this->sleevetype = "";
        }

        Shirt(string color, string sleevetype)
        {
            this->color = color;
            this->sleevetype = sleevetype;
        }

        string get_color() const
        {
            return this->color;
        }

        void set_color(string color)
        {
            this->color = color;
        }

        string get_sleevetype() const
        {
            return this->sleevetype;
        }

        void set_sleevetype(string sleevetype)
        {
            this->sleevetype = sleevetype;
        }


        ~Shirt()
        {

        }

};